crates/rustc_utils/src/mir/place.rs

```diff
@@ -202,69 +202,54 @@ impl<'tcx> PlaceExt<'tcx> for Place<'tcx> {
     def_id: DefId,
   ) -> HashMap<RegionVid, Vec<(Place<'tcx>, Mutability)>> {
     let ty = self.ty(body.local_decls(), tcx).ty;
-    let mut region_collector = CollectRegions {
+    let mut region_collector = RegionVisitor::<RegionMemberCollector>::new(
       tcx,
       def_id,
-      local: self.local,
-      place_stack: self.projection.to_vec(),
-      ty_stack: Vec::new(),
-      regions: HashMap::default(),
-      places: None,
-      types: None,
-      stop_at: if
+      *self,
+      if
       /*shallow*/
       false {
         StoppingCondition::AfterRefs
       } else {
         StoppingCondition::None
       },
-    };
+    );
     region_collector.visit_ty(ty);
-    region_collector.regions
+    region_collector.into_inner().0
   }
 
   fn interior_places(
     &self,
     tcx: TyCtxt<'tcx>,
     body: &Body<'tcx>,
     def_id: DefId,
-  ) -> Vec<Place<'tcx>> {
+  ) -> HashSet<Place<'tcx>> {
```

Comment:
You can use the `PlaceSet` alias I define elsewhere.

Reply:
Sadly not. `PlaceSet` is defined in flowistry.